### PR TITLE
Maaz/responsive

### PR DESCRIPTION
**Pull request by MaazSiddiqi**
Saving my responsive changes

### CODE REVIEW

File: src/Pages/AboutUs/Aboutus.css [hunk 1]
```diff
@@ -188,53 +201,40 @@
...
+  width: 96vw;
+  padding: 4rem 2vw;
+
   display: flex;
   flex-direction: row;
   justify-content: center;
-  pointer-events: none;
+
   font-family: "Red Hat Display", sans-serif;
-  padding: 0% 50px 0% 50px;
```
Comment (LucasHartmanWestern) [inline src/Pages/AboutUs/Aboutus.css:196]: 0 does not need % next to it

File: src/Pages/AboutUs/Aboutus.js [hunk 2]
```diff
@@ -52,46 +51,42 @@ export default function Aboutus() {
...
+                <br />
+                There are many benefits that come with collaborating with our
+                association! Partnering with WDS means that we will share and
+                promote your organization&#x27;s marketing material to the
+                Western Engineering &amp; Computer Science and greater Western
+                community. You&#x27;ll also have opportunities to directly
+                engage with the student population through sending
+                representatives from your company, and/or through other
+                collaboration formats. This is great for recruitment purposes!
+              </p>
```
Comment (LucasHartmanWestern) [inline src/Pages/AboutUs/Aboutus.js]: Text this long should be stored in a separate file, not within the HTML itself

File: src/Pages/Contact/Contact.css [hunk 3]
```diff
@@ -1,109 +1,114 @@
...
+  justify-content: center;
+  align-items: flex-start;
 
-  padding: 250px 350px;
+  color: rgb(51, 50, 50);
 }
 
 .Contact-1 {
-  font-size: 60px;
+  padding: 2rem 0rem;
```
Comment (LucasHartmanWestern) [inline src/Pages/Contact/Contact.css]: 0 does not need rem next to it

File: src/Pages/Contact/Contact.css [hunk 4]
```diff
@@ -1,109 +1,114 @@
...
 .Contact-check {
-  padding-top: 2%;
-  justify-content: start !important;
-  width: 100%;
+  font-weight: 500;
 }
 
 .Contact-Socials {
+  width: 100%;
+  padding: 0rem 1rem;
```
Comment (LucasHartmanWestern) [inline src/Pages/Contact/Contact.css]: 0 does not need rem next to it

File: src/Pages/Contact/Contact.css [hunk 5]
```diff
@@ -1,109 +1,114 @@
...
-  width: max-content;
-  padding: 7% 0%;
-  justify-items: center;
+  justify-content: space-around;
+  gap: 0.5rem;
 }
 
 .Contact-Socials > div {
-  padding: 0.7rem 1.2rem;
+  padding: 0.3rem 0rem;
```
Comment (LucasHartmanWestern) [inline src/Pages/Contact/Contact.css]: 0 does not need rem next to it

File: src/Pages/Contact/Contact.css [hunk 6]
```diff
@@ -1,109 +1,114 @@
...
-  border-radius: 3px;
-  border-width: 0px;
-  min-height: 40px;
+  min-height: 25px;
   margin-top: 10px;
-  border: 1px solid rgb(214, 214, 214);
+  padding: 0.5rem 1rem;
+
+  border-radius: 25px;
+  border-width: 0px;
```
Comment (LucasHartmanWestern) [inline src/Pages/Contact/Contact.css]: 0 does not need px next to it

File: src/Pages/Home/Home.css [hunk 7]
```diff
@@ -10,25 +9,19 @@
...
 .hero {
-  width: 100%;
-  padding:15% 5%;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-}
+  width: 80vw;
+  height: 100vh;
+  padding: 0px 10vw;
```
Comment (LucasHartmanWestern) [inline src/Pages/Home/Home.css]: 0 does not need px next to it

File: src/Pages/Home/Home.css [hunk 8]
```diff
@@ -103,16 +91,16 @@
...
 
   font-size: 18px;
   color: rgb(51, 50, 50);
-  background-color: rgb(241, 241, 241);
+  background-color: #eeeeee;
 }
 
 .what-is {
-  padding: 4rem 0rem;
+  padding: 2rem 0px;
```
Comment (LucasHartmanWestern) [inline src/Pages/Home/Home.css]: 0 does not need px next to it

File: src/Pages/Home/Home.css [hunk 9]
```diff
@@ -126,32 +114,23 @@
...
-  align-items: center;
-}
-
-.what-is-point > * {
-  padding: 0rem 0.4rem;
-}
-
 .details-container {
-  padding: 2rem 0rem;
+  padding: 1.3rem 0px;
```
Comment (LucasHartmanWestern) [inline src/Pages/Home/Home.css]: 0 does not need px next to it

File: src/Pages/Home/Home.css [hunk 10]
```diff
@@ -229,43 +207,61 @@
...
+  .details-left {
+    width: 500px;
   }
-  .hero-container{
+}
+
+@media screen and (max-width: 1050px) {
+  .hero {
+    width: 98vw;
+    padding: 0px 1vw;
```
Comment (LucasHartmanWestern) [inline src/Pages/Home/Home.css]: 0 does not need px next to it

File: src/Pages/Home/Home.css [hunk 11]
```diff
@@ -229,43 +207,61 @@
...
-    gap:1rem;
+    margin-top: 60px;
+
+    font-size: 60px;
+    line-height: 3rem;
+  }
+
+  .pic {
+    padding-top: 50px;
+    padding-left: 0px;
```
Comment (LucasHartmanWestern) [inline src/Pages/Home/Home.css]: 0 does not need px next to it

**Review by LucasHartmanWestern — CHANGES_REQUESTED** [review]
Overall looks good, just a couple of minor nitpicks

File: src/Pages/AboutUs/Aboutus.css (hunk 1 above)
Comment (MaazSiddiqi) [reply src/Pages/AboutUs/Aboutus.css:196]: deleted line lol

File: src/Pages/AboutUs/Aboutus.js (hunk 2 above)
Comment (MaazSiddiqi) [reply src/Pages/AboutUs/Aboutus.js]: agreed, but for now i've just stored it in a constant in the same file. felt weird making a whole file just for that one constant, and doesn't rlly fit with members data yk.

**Comment by LucasHartmanWestern** [comment]
lgtm

**Review by LucasHartmanWestern — APPROVED** [review]
lgtm

**Pull request merged**